src/layouts/blank.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div>
-    <VMigrationNotice v-show="isReferredFromCc" />
+    <VMigrationNotice v-if="isReferredFromCc" />
```

Comment:
Would this cause client-server mismatch? Often when you use `v-if` and the server state is different from the client state (i.e., something depends on the client state such as localStorage), the mismatch will cause 'hydration bail' and full re-rendering on the client.

Reply:
Ah, I thought I remembered there being a good reason for it being `v-show`. I'll put it back.

Reply:
We can move it to `v-if` if we want when we implement the ui state cookie.